aiida_mlip/calculations/base.py

```diff
@@ -301,6 +303,9 @@ def _add_model_to_cmdline(
         cmd_line : dict
             Dictionary containing the cmd line keys.
 
+        folder : ~aiida.common.folders.Folder
+            An `aiida.common.folders.Folder` to temporarily write files on disk.
```

Comment:
I still think this is misleading if it's not temporary, unless I misunderstood our discussion?

Reply:
no I forgot to change